code/__HELPERS/unsorted.dm

```diff
@@ -1192,12 +1192,10 @@ var/global/list/common_tools = list(
 	return 0
 
 /proc/is_hot(obj/item/W as obj)
```

Comment:
This proc is gross too, but don't worry about refactoring it.

Reply:
I wasn't planning on it 😄

Reply:
Too hot to touch!